### ISSUE TITLE
adjacent-overload-signatures incorrect with one-liners

### ISSUE THREAD

**Issue by MrAndersen1** [palantir/tslint#2482]
### Bug Report

- __TSLint version__: 5.0.0
- __TypeScript version__: 2.3.0-dev.20170329
- __Running TSLint via__: (pick one) CLI and VSCode

#### TypeScript code being linted

```ts
class Testclass {
	private _test: number;
	set test(v: number) { this._test = v; };
	get test(): number { return this._test; };
}
```

with `tslint.json` configuration:

```json
{
    "extends": "tslint:latest",
    "rules": {}
}
```

#### Actual behavior
ERROR: test.ts[6, 2]: All 'test' signatures should be adjacent
#### Expected behavior
No error.

I suppose it could rather be an error with getters and setters depending on how it should behave.

**Comment by andy-hanson** [comment]
Quickfix: Remove the semicolons, they aren't needed. Although I updated the rule to ignore them since warning on extra semicolons is a different rule.

**Comment by MrAndersen1** [comment]
@andy-hanson Hm, I was sure I had them due to some other tslint warning but when I remove them I get none. I was probably using an old version then.
